packages/react-ui/src/localization/index.js

```diff
@@ -0,0 +1,11 @@
+import en from './en';
+
+import { flattenMessages } from './localeUtils';
+
+const enFlat = flattenMessages(en);
+
+export const messages = {
```

Comment:
As we might eventually add localization in other parts (as error messages or similar), I feel this could probably go into core, what do you think?

Reply:
cc/ @zbigg